### PR TITLE
Global Context: User Info

### PR DESCRIPTION
**Pull request by Dulaya**

### CODE REVIEW

**Review by raynesmike — APPROVED** [review]

File: mobile/src/context/user/user.reducer.ts [hunk 1]
```diff
@@ -1,25 +1,40 @@
...
+      return {
+        ...prevState,
+        address: action.address,
+      }
+    case EUserAction.SET_PHONE:
+      return {
+        ...prevState,
+        phone: action.phone,
+      }
+    case EUserAction.SET_USER_INFO:
```
Comment (mshago) [inline mobile/src/context/user/user.reducer.ts:30]: I don't know if the individual sets are necessary

Comment (Dulaya) [reply mobile/src/context/user/user.reducer.ts:30]: I did them incase if the user need to change their only name in the profile page.

File: mobile/src/context/user/user.state.ts [hunk 2]
```diff
@@ -1,7 +1,14 @@
...
+import { TUserState } from '../user';
 
-export const authInitialState:TAuthState = {
-  isSignOut: false,
-  token: null,
-  isLoading: true,
+export const userInitialState: TUserState = {
+  name: '',
+  email: '',
+  picture: '',
```
Comment (mshago) [inline mobile/src/context/user/user.state.ts:6]: you are using picture here but in the user state type is named image

Comment (Dulaya) [reply mobile/src/context/user/user.state.ts:6]: Totally forgot about that when I changed from names from 'picture' to 'image'.

**Review by Josetf2396 — APPROVED** [review]

**Review by lasharela — APPROVED** [review]

**Review by mshago — APPROVED** [review]

**Pull request merged**